src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx

```diff
@@ -119,7 +119,7 @@ const IfoCardTokens: React.FC<IfoCardTokensProps> = ({
             />
           </TokenSection>
           <TokenSection img={tokenImage}>
-            <Label> {t(hasClaimed ? '%symbol% received' : '%symbol% to received', { symbol: currency.symbol })}</Label>
+            <Label> {t(hasClaimed ? '%symbol% received' : '%symbol% to receive', { symbol: token.symbol })}</Label>
```

Comment:
My bad sorry :(

Reply:
@memoyil Its fine, easy to miss when you edit 100+ files 😄